Formula/elm-format.rb

```diff
@@ -42,8 +43,8 @@ def install
 
     # Directly running `cabal v2-install` fails: Invalid file name in tar archive: "avh4-lib-0.0.0.1/../"
     # Instead, we can use the upstream's build.sh script, which utilizes the Shake build system.
```

Comment:
maybe remove this line?

Reply:
The build still fails when trying ot use `cabal v2-install`. Unless I'm misunderstanding which line you're referring to.

Reply:
Oh, i see, i thought we are using `hpack` for the build now. never mind then :)

Reply:
The build script uses `hpack` now 👍